content/en/monitors/create/configuration.md

```diff
@@ -120,18 +120,50 @@ See the documentation for [process check][1], [integration check][2], and [custo
 
 #### No data
 
+Notifications for missing data are useful if you expect a metric to always be reporting data under normal circumstances. For example, if a host with the Agent must be up continuously, you can expect the metric `system.cpu.idle` to always report data. For this case, you should enable notifications for missing data. The sections below explain how to accomplish this with each option.
+
+There are currently two ways to deal with missing data:
+- Metric based monitors use the limited `Notify no data` option
+- The `On missing data` option is supoprted by APM Trace Analytics, Audit Logs, CI Pipelines, Error Tracking, Events, Logs, and RUM Monitors
+
+{{< tabs >}}
+{{% tab "Metric based monitors" %}}
+
 `Do not notify` if data is missing or `Notify` if data is missing for more than `N` minutes.
 
-Notifications for missing data are useful if you expect a metric to always be reporting data under normal circumstances. For example, if a host with the Agent must be up continuously, you can expect the metric `system.cpu.idle` to always report data. For this case, you should enable notifications for missing data.
+This option works in a binary fashion. You either get notified for data being missing, or not. The notification occurs when no data was received during a configured time window.
 
 **Note**: It is recommended that you set the missing data window to at least two times the evaluation period.
 
-Alternatively, if you are monitoring a metric over an auto-scaling group of hosts that stop and start automatically, notifying for no data would produce a lot of notifications. For this case, you should not enable notifications for missing data. This option does not work if it is enabled at a time when data has not been reporting for a long period.
+If you are monitoring a metric over an auto-scaling group of hosts that stop and start automatically, notifying for no data would produce a lot of notifications. For this case, you should not enable notifications for missing data. This option does not work if it is enabled at a time when data has not been reporting for a long period.
 
 ##### Grouping
 
 For a monitor that does not notify on missing data, if a group does not report data, the monitor skips evaluations and eventually drops the group. During this period, the bar in the results page stays green. When there is data and groups start reporting again, the green bar shows an OK status and backfills to make it look like there was no interruption.
 
+{{% /tab %}}
+
+{{% tab "Other monitor types" %}}
+
+{{< img src="/monitors/create/on_missing_data.png" alt="No Data Options" style="width:90%;">}}
+
+If data is missing for `N` minutes: `Evaluate as zero` / `Show last known status`, `Show NO DATA`, `Show NO DATA and notify`, or `Show OK`.
```

Comment:
Should this be a bullet point list instead of listed on the same line?

Reply:
And should we put it above the image so that the image illustrate the your list?

Also you might want to reduce the image size (like put a smaller %)